lib/utils/httpDate.js

```diff
@@ -11,9 +11,12 @@
  * @returns  {String}     formatted dated object
  */
 module.exports = function httpDate(now) {
```

Comment:
Should probably call this out as an API change. `now` will no longer be set to the current `Date` if something a variable is passed in that is falsey.